plugins/inputs/win_wmi/win_wmi.go

```diff
@@ -48,73 +47,64 @@ func oleInt64(item *ole.IDispatch, prop string) (int64, error) {
 	if err != nil {
 		return 0, err
 	}
-	defer func() { _ = v.Clear() }()
+	defer v.Clear()
 
-	i := v.Val
-	return i, nil
+	return v.Val, nil
 }
 
 // Init function
 func (s *Wmi) Init() error {
-	if err := CompileInputs(s); err != nil {
-		return err
-	}
-	return nil
-}
-
-// Description function
-func (s *Wmi) Description() string {
-	return "returns WMI query results as metrics"
+	return compileInputs(s)
 }
 
 // SampleConfig function
 func (s *Wmi) SampleConfig() string {
 	return sampleConfig
 }
 
-func CompileInputs(s *Wmi) error {
-	BuildWqlStatements(s)
-	if err := CompileTagFilters(s); err != nil {
+func compileInputs(s *Wmi) error {
+	buildWqlStatements(s)
+	if err := compileTagFilters(s); err != nil {
 		return err
 	}
 	return nil
```

Comment:
I think this can also be compacted?
```suggestion
	return compileTagFilters(s)
```

Reply:
Done.